packages/editor/src/client/routes/userList.tsx

```diff
@@ -119,51 +138,90 @@ export function UserList() {
-        <Column width={400} align="left" resizable>
-          <HeaderCell>{t('description')}</HeaderCell>
-          <Cell dataKey="email" />
-        </Column>
-        <Column width={100} align="center" fixed="right">
-          <HeaderCell>{t('action')}</HeaderCell>
-          <Cell style={{padding: '6px 0'}}>
-            {(rowData: FullUserFragment) => (
-              <>
-                <IconButton
-                  icon={<Icon icon="key" />}
-                  circle
-                  size="sm"
-                  style={{marginLeft: '5px'}}
-                  onClick={e => {
-                    setCurrentUser(rowData)
-                    setIsResetUserPasswordOpen(true)
-                  }}
-                />
-                <IconButton
-                  icon={<Icon icon="trash" />}
-                  circle
-                  size="sm"
-                  style={{marginLeft: '5px'}}
-                  onClick={() => {
-                    setConfirmationDialogOpen(true)
-                    setCurrentUser(rowData)
-                  }}
-                />
-              </>
-            )}
-          </Cell>
-        </Column>
-      </Table>
+      <div
+        style={{
+          display: 'flex',
+          flexFlow: 'column',
+          marginTop: '20px'
+        }}>
+        <Table
+          autoHeight={true}
+          style={{flex: 1}}
+          loading={isLoading}
+          data={users}
+          sortColumn={sortField}
+          sortType={sortOrder}
+          onSortColumn={(sortColumn, sortType) => {
+            setSortOrder(sortType)
+            setSortField(sortColumn)
+          }}>
+          <Column width={200} align="left" resizable sortable>
+            <HeaderCell>{t('userList.overview.createdAt')}</HeaderCell>
+            <Cell dataKey="createdAt">
+              {({createdAt}: FullUserFragment) => new Date(createdAt).toDateString()}
+            </Cell>
+          </Column>
+          <Column width={200} align="left" resizable sortable>
+            <HeaderCell>{t('userList.overview.modifiedAt')}</HeaderCell>
+            <Cell dataKey="modifiedAt">
+              {({modifiedAt}: FullUserFragment) => new Date(modifiedAt).toDateString()}
+            </Cell>
+          </Column>
+          <Column width={200} align="left" resizable sortable>
+            <HeaderCell>{t('userList.overview.name')}</HeaderCell>
+            <Cell dataKey={'name'}>
+              {(rowData: FullUserFragment) => (
+                <Link route={UserEditRoute.create({id: rowData.id})}>
+                  {rowData.name || t('userList.overview.unknown')}
+                </Link>
+              )}
+            </Cell>
+          </Column>
+          <Column width={400} align="left" resizable>
+            <HeaderCell>{t('email')}</HeaderCell>
+            <Cell dataKey="email" />
+          </Column>
+          <Column width={100} align="center" fixed="right">
+            <HeaderCell>{t('action')}</HeaderCell>
```

Comment:
I would rather use here `userList.overview.action` in order to be consistent with the rest